### PR TITLE
Update Compiler API content in release note 2201.2.0

### PR DESCRIPTION
**Pull request by sanjana**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by praneesha — APPROVED** [review]

**Review by dulajdilshan — APPROVED** [review]
LGTM

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 1]
```diff
@@ -190,12 +182,20 @@ digraph "org/package:0.1.0" {
...
+
 ##### Ballerina Shell
 
 ##### Ballerina Update Tool
 
 #### Improvements
 
+##### Compiler API
+
+- Added semantic API support for the `client resource access` action
```
Comment (MaryamZi) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:197]:
```suggestion
- Added semantic API support for the client resource access action
```

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:197]: @MaryamZi - It is fine to highlight the action name within backticks to separate it out from the normal text. We have followed the same elsewhere also.

Comment (MaryamZi) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:197]: We only emphasize keywords, variable names, etc. Which category does this come under?

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:197]: Action names are also like variable names and also this is done just for more clarity. Do we have a strong reason to not do it?

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:197]: Sorry, I was referring to code actions. We will not need backticks as this is a common action.

Comment (sanjana) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: Has been fixed.

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**